src/main/java/de/tum/in/www1/artemis/service/metis/PostService.java

```diff
@@ -62,19 +60,22 @@ public class PostService extends PostingService {
 
     private final PlagiarismCaseService plagiarismCaseService;
 
+    private final AuthorizationCheckService authorizationCheckService;
+
     private final PostSimilarityComparisonStrategy postContentCompareStrategy;
 
     protected PostService(CourseRepository courseRepository, AuthorizationCheckService authorizationCheckService, UserRepository userRepository, PostRepository postRepository,
             ExerciseRepository exerciseRepository, LectureRepository lectureRepository, GroupNotificationService groupNotificationService,
             PostSimilarityComparisonStrategy postContentCompareStrategy, SimpMessageSendingOperations messagingTemplate, PlagiarismCaseService plagiarismCaseService,
-            PlagiarismCaseRepository plagiarismCaseRepository) {
+            PlagiarismCaseRepository plagiarismCaseRepository, AuthorizationCheckService authorizationCheckService1) {
```

Comment:
```suggestion
            PlagiarismCaseRepository plagiarismCaseRepository, AuthorizationCheckService authorizationCheckService) {
```

Reply:
You can remove that parameter, it's already the second constructor parameter.

Reply:
done!